services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -67,7 +67,7 @@
 SUPPORTED_TRAEFIK_LOG_LEVELS: set[str] = {"info", "debug", "warn", "error"}
 
 PlacementConstraintStr = constr(
-    strip_whitespace=True, regex=r"^[a-zA-Z0-9. ]*(!=|==){1}[a-zA-Z0-9. ]*$"
+    strip_whitespace=True, regex=r"^[a-zA-Z0-9_. ]*(!=|==){1}[a-zA-Z0-9. ]*$"
```

Comment:
may I suggest it like this?
```suggestion
    strip_whitespace=True, regex=r"^[a-zA-Z0-9_. ]*(!=|==){1}[a-zA-Z0-9_. ]*$"
```